.github/workflows/ci.yml

```diff
@@ -0,0 +1,61 @@
+name: Nginx image creator
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+
+  create-and-push-image:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+      packages: write
+      attestations: write
+      id-token: write
+
+    steps:
+    - name: CHECKOUT
+      uses: actions/checkout@v3
+
+    - name: INSTALL Node
+      working-directory: ./node
+      run: npm install
```

Comment:
Usually it is better to use `npm ci` - https://builtin.com/software-engineering-perspectives/npm-ci-vs-npm-install